### PR TITLE
feat: Implement feature flag for quick create chat discussion feature - EXO-60940

### PR DESCRIPTION
**Pull request by sofyenne**
Implement feature flag for quick create chat discussion feature

### CODE REVIEW

File: services/src/main/resources/conf/portal/configuration.xml [hunk 1]
```diff
@@ -34,7 +34,17 @@
...
+    <key>QuickCreateChatDiscussionFeatureProperties</key>
+    <type>org.exoplatform.container.ExtendedPropertyConfigurator</type>
+    <init-params>
+      <properties-param>
+        <name>QuickCreateChatDiscussionFeatureProperties</name>
+        <description>Quick Create Chat Discussion Feature enablement flag</description>
+        <property name="exo.feature.quickCreateChatDiscussion.enabled" value="${exo.feature.quickCreateChatDiscussion.enabled:true}"/>
+      </properties-param>
+    </init-params>
+    </component>
```
Comment (azayati) [inline services/src/main/resources/conf/portal/configuration.xml]:
```suggestion
    <component>
      <key>QuickCreateChatDiscussionFeatureProperties</key>
      <type>org.exoplatform.container.ExtendedPropertyConfigurator</type>
      <init-params>
        <properties-param>
          <name>QuickCreateChatDiscussionFeatureProperties</name>
          <description>Quick Create Chat Discussion Feature enablement flag</description>
          <property name="exo.feature.quickCreateChatDiscussion.enabled" value="${exo.feature.quickCreateChatDiscussion.enabled:false}"/>
        </properties-param>
      </init-params>
    </component>
```

> Default value: false

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: services/src/main/resources/conf/portal/configuration.xml (hunk 1 above)
Comment (sofyenne) [reply services/src/main/resources/conf/portal/configuration.xml]: PR updated !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**